packages/invoices/src/InboundShipment/DetailView/Footer.tsx

```diff
@@ -103,8 +107,15 @@ export const Footer: FC<InboundDetailFooterProps> = ({ draft, save }) => {
                     variant="contained"
                     color="secondary"
                     onClick={async () => {
-                      success('Saved invoice! 🥳 ')();
-                      save();
+                      update(
+                        {
+                          onHold: onHoldBuffer,
+                          status: InvoiceNodeStatus.Verified,
+                        },
+                        {
+                          onSuccess: success('Saved invoice! 🥳'),
```

Comment:
Not sure if a toast here or something else, so haven't translated. Was thinking to have a blocking spinner when saving?

Reply:
a blocking spinner is probably best, otherwise the update might overwrite local changes if it is a slow update.
Wondered about just disabling the save button while updating and/or having a spinner icon on the button while updating, but think we need to block